### DISCUSSION TITLE
does using self-hosted runners add to action minutes usage?

### DISCUSSION THREAD

**Discussion by vivekthangathurai** [community/community#26444] [Actions]
<p>I just want to know if we are using self-hosted runners to our pipeline. does it contribute to the actions minutes in our plan?</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/vivekthangathurai">@vivekthangathurai</a> ,</p>
<p>It’s free to use self-hosted runner with GitHub Actions, will NOT increase the actions minutes for billing.</p>
<p>It’s recomended to use self-hosted runner in private repository, it’s possible to incur a fee for extra data storage(eg: packages, artifacts) and data transfer above the default limit. More details please refer to the official doc <a href="https://help.github.com/en/github/setting-up-and-managing-billing-and-payments-on-github/about-billing-for-github-actions" rel="nofollow noopener">here</a>.</p>
<p>Thanks.</p>
